### ISSUE TITLE
kmscon with systemd and startx broken

### ISSUE THREAD

**Issue by vodik** [kmscon/kmscon#65]
On archlinux, I setup kmscon to replace agetty by creating a symlink in `/etc/systemd/system/` between `autovt@.service` and `kmsconvt@.service`. This works great as systemd now automatically starts kmscon instead of agetty.

Now typically I use startx in place on a tty. So when I launch xmonad through startx on tty1, it launches X on tty1 instead of creating allocating new tty. This is necessary to have startx work nicely with loginctl. If startx instead grabs a new tty, according to systemd, the login is no longer considered active.

So when running startx on tty1, I essentially run `startx -- vt0`

However it seems like running kmscon prevents access to the underlying virtual terminal. Running the above command yields the following error:

```
Fatal server error:
xf86OpenConsole: VT_ACTIVATE failed: No such device or address
```

Basically, afaik, X can't draw on top of kmscon like it could with agetty.

**Comment by kdj0c** [comment]
You can now use kmscon-launch-gui, to start a graphic session.
